modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,694 @@
+
+For information on the **Spellchecker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[Spellchecker Pro].
+
+=== Spelling Autocorrect 1.0.1
+
+The {productname} 6.3 release includes an accompanying release of the **Spelling Autocorrect** premium plugin.
+
+**Spelling Autocorrect** 1.0.1 includes one fix:
+
+==== Fixed format changes at the start of sentences when auto-capitalization was on
+
+When **Spelling Autocorrect** was active and had Capitalization enabled (ie had `autocorrect_capitalize: true` set), any inline formatting applied at the beginning of a new sentence, after a full stop (period) but before the start of the word, would be lost when the word was auto-capitalized. Instead, the format at the end of the previous sentence was incorrectly applied. 
+
+This update to the **Spelling Autocorrect** premium plugin removed redundant ‘pre-text’ punctuation, before the word at the start of the sentence being capitalized, from start and replacement of text-pattern.
+
+As a consequence, formatting is now retained when the word is capitalized, as expected.
+
+[[accompanying-premium-skins-and-icon-packs-changes]]
+== Accompanying Premium Skins and Icon Packs changes
+
+The {productname} 6.3 release includes an accompanying release of the **Premium Skins and Icon Packs**.
+
+=== Premium Skins and Icon Packs
+
+The **Premium Skins and Icon Packs** release includes the following updates:
+
+The **Premium Skins and Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 6.3 skin, Oxide.
+
+For information on using premium skins and icon packs, see: xref:premium-skins-and-icons.adoc[Premium Skins and Icon Packs].
+
+[[improvements]]
+== Improvements
+
+{productname} 6.3 also includes the following improvements
+
+=== The Text color and Background color menus now display a check mark denoting the currently selected color
+
+Previously, the color swatches displayed when the Text color and Background color menus were opened gave no indication as to whether or not the Text color or Background color of the current selection was one of the colors displayed in these menus.
+
+With this update, when the selected text’s current Text color or Background color matches one of the colors available from the appropriate color swatch menu, a check mark (the ✔️ character, also known as a tick) displays over that color.
+
+=== Text color and Background color toolbar menus now pre-select the color for the selected text or insertion point position.
+
+{productname} 6.3 included an improvement to the Text color or Background color menus that allows the user to identify the pre-populated color for the selected-text: a ✔️ character displayed over the color that matches the currently selected Text color or Background color.
```

Comment:
```suggestion
{productname} 6.3 included an improvement to the Text color or Background color menus that allows the user to identify the pre-populated color for the selected-text: a tick/checkmark icon displayed over the color that matches the currently selected Text color or Background color.
```
The image used to overlay the selected color is an svg icon of a checkmark, not a character. Either tick or checkmark is fine to use.

Reply:
```suggestion
{productname} 6.3 included an improvement to the Text color or Background color menus that allows the user to identify the pre-populated color for the selected-text: a checkmark (also known as a tick or tickmark) displayed over the color that matches the currently selected Text color or Background color.
```